drivers/hmis/app/views/hmis_client/services/show.haml

```diff
@@ -0,0 +1,34 @@
+- title = @service.service_name
+- content_for :modal_title, title
+= content_for :crumbs do
+  = link_to appropriate_client_path(@client) do
+    &laquo; Client
+
+.well
+  .d-flex
+    .service
+      %h3 Service Details
+      %dl
+        %dt Project
+        %dd= @service.enrollment.project.warehouse_project.name(current_user)
+        %dt Date Provided
+        %dd= @service.DateProvided
+        %dt Created
+        %dd= @service.DateCreated
+        %dt Updated
+        %dd= @service.DateUpdated
```

Comment:
We should include, if present:
`@service.FAAmount`
`@service.FAStartDate`
`@service.FAEndDate`

We may want to use this modal for HUD Services, too.. as far as I can tell there is no way currently to see the FA fields on a HUD Service?

The HMIS can also support recording custom fields on HUD services. So it seems reasonable we could use this detail modal for both record types.

Reply:
It might be slightly painful, but you can see these values for HUD services under the little HMIS button.

<img width="1400" alt="Screenshot 2024-07-01 at 11 42 22 AM" src="https://github.com/greenriver/hmis-warehouse/assets/1346876/de88ec87-e2b1-4a23-833a-f7300f8bbfb1">